bit9_connector.py

```diff
@@ -324,15 +379,16 @@ def _block_hash(self, param):
             return action_result.get_status()
 
         if len(rules) > 1:
-            return action_result.set_status(phantom.APP_ERROR, "More than one rule matched for the hash. This is treated as an Error.")
+            return action_result.set_status(phantom.APP_ERROR,
+                                            "More than one rule matched for the hash. This is treated as an Error.")
 
         file_rule = {}
 
         if rules:
             self.save_progress("Got Rule for file")
             file_rule = rules[0]
 
-        if (catalog_found) and ('id' in catalog_found):
+        if catalog_found and ('id' in catalog_found):
```

Comment:
This can be reverted.

Reply:
this could be but getting warning on IDE so removed it.